tests/system-test/email_recipient_list.dat

```diff
@@ -0,0 +1 @@
+ci_report@lightbitslabs.com
```

Comment:
@elada-lb how does this work? In generl, the result of the PR check should be emailed to the user who initiated it.